src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/events/_events.svelte

```diff
@@ -5,6 +5,13 @@
 
   export let history: History;
   export let eventFormat: EventFormat;
+  export let eventType;
+
+  $: events = history.events.filter((event) => {
+    if (eventType && !String(event.eventType).startsWith(eventType))
```

Comment:
is this for performance or why `startsWith` and not `includes` ?

Reply:
I used the same logic as the workflow filters, but would you say ‘includes’ will be better in this case since users will be able to type in any part of the phrase they are searching for?

Reply:
@josephriosIO for UX - yes. For perf - no

Usually i would prioritize UX, unless there is a noticeable perf decrease (in which case this affects UX too)